### DISCUSSION TITLE
v0.0.1

### DISCUSSION THREAD

**Discussion by aecyia** [aecyia/Celeste#3] [Announcements]
## What's Changed
* v0.0.1 by @VMPYRC in https://github.com/aecyia/Celeste/pull/1

## New Contributors
* @VMPYRC made their first contribution in https://github.com/aecyia/Celeste/pull/1

**Full Changelog**: https://github.com/aecyia/Celeste/compare/v0.0.0...v0.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/aecyia/Celeste/releases/tag/v0.0.1'>v0.0.1</a>.</em>

**Comment by VMPYRC** [comment]
# Vanity Images

![EnterCAS](https://github.com/user-attachments/assets/2f8b5f61-2bae-4ffb-9a12-7b90f49c66d9)
![CAS](https://github.com/user-attachments/assets/72d59081-4c87-4eb5-9cf2-a32885797f04)
![CreateAPet_AdvancedMode](https://github.com/user-attachments/assets/5a2cf58c-5f7d-406e-bf07-258112260bad)

![BuildBuy_CreateAStyle](https://github.com/user-attachments/assets/8fc2dbd1-ae88-4b70-9c88-474162da4b06)
![BuildBuyMode](https://github.com/user-attachments/assets/9681399a-9ba3-4e0b-9c22-7d22959482fa)

![PieMenu](https://github.com/user-attachments/assets/3a1435c6-b6da-4599-bd95-681e4ffc3ed6)
